.github/SECURITY.md

```diff
@@ -0,0 +1,23 @@
+# Security Policy
+
+## Supported Versions
+
+The latest patch version of the `2.x` release series is supported for security updates.
```

Comment:
This seems like a bit of a conflicting statement. Assuming that we would add security updates a patch versions, this would mean that the latest **patch** version is not "supported", but rather "replaced".
Shouldn't it rather be the latest **minor** version being supported?

Reply:
Done